engine/runtime-with-instruments/src/test/scala/org/enso/interpreter/test/instrument/RuntimeErrorsTest.scala

```diff
@@ -1964,4 +1977,106 @@ class RuntimeErrorsTest
     context.consumeOut shouldEqual List("MyError")
   }
 
+  it should "cache dataflow errors" in {
+    val contextId  = UUID.randomUUID()
+    val requestId  = UUID.randomUUID()
+    val moduleName = "Enso_Test.Test.Main"
+    val newline    = "\n" // was: System.lineSeparator()
+
+    val metadata   = new Metadata
+    val xId        = metadata.addItem(111, 79)
+    val mainResId  = metadata.addItem(195, 1)
+    val mainRes1Id = metadata.addItem(209, 1)
+
+    val code =
+      """import Standard.Base.Error.Error
+        |import Standard.Base.Errors.Illegal_Argument.Illegal_Argument
+        |
+        |main =
+        |    x = Error.throw (Illegal_Argument.Error "The operation failed due to some reason.")
+        |    x
+        |""".stripMargin.linesIterator.mkString("\n")
```

Comment:
Should the code use `mkString(newline)`? Or what is the point of defining `newline` variable, when it is not used?

Reply:
`newline` is used in other parts in string interpolation. Having said that, it seems `linesIterator.mkString("\n")` is spurious in this and other locations.

Reply:
Reverted my cleanup of `linesIterator.mkString("\n")`. I think Windows doesn't like it.